packages/oidc-js/src/utils/op-config.ts

```diff
@@ -198,9 +198,13 @@ export const initOPConfiguration = (
     }
 
     const serverHost = requestParams.serverOrigin;
+    let wellKnownEndpoint = serverHost + SERVICE_RESOURCES.wellKnown;
+    if (requestParams?.endpoints?.wellKnown) {
+        wellKnownEndpoint = requestParams?.endpoints?.wellKnown;
```

Comment:
Null will be handled inside if. So no need to add it again.
```suggestion
        wellKnownEndpoint = requestParams.endpoints.wellKnown;
```